### PR TITLE
[#2558] future proof scripts/brotli.js

### PR DESCRIPTION
**Pull request by drkstr101**
Closes 

## ✅ Pull Request Checklist:

- [ x] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [n/a] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [n/a ] Filled out test instructions.
- [n/a ] Updated documentation (if it already exists for this component).
- [n/a ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

https://github.com/adobe/react-spectrum/issues/2558

A very minor and optional update to use dynamic import for `p-queue` construction.

reasons: better standards support (arguably)

Feel free to reject without comment, or discuss.

Cheers!

### CODE REVIEW

**Comment by drkstr101** [comment]
CLA is signed. It must take a bit to sync. Also see rebase/resubmit (save-without-formatting).

File: scripts/brotli.js [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+ * the License is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
 
 const fs = require('fs');
 const glob = require('fast-glob');
-const PQueue = require('p-queue').default;
 const zlib = require('zlib');
+// const PQueue = require('p-queue').default;
```
Comment (snowystinger) [inline scripts/brotli.js:16]: no reason to keep this line around with your change right?

**Review by snowystinger — COMMENTED** [review]
Great, thanks for fixing this for node 16. I think your editor has "fixed" a couple of things it shouldn't have. Would you mind putting those lines back? Specifically the license :)

**Comment by snowystinger** [comment]
In order to get the CLA to take hold you will need to close and reopen the PR. You don't need to open a new one, just reopen this one after you close it.

File: scripts/brotli.js (hunk 1 above)
Comment (drkstr101) [reply scripts/brotli.js:16]: None at all. I will remove and resubmit.

Cheers!

**Comment by drkstr101** [comment]
Close/reopen as per instructions.

**Comment by drkstr101** [comment]
Hmm, maybe I didn't finish some steps. I am still getting links to sign in MR.  I'll check the confirmation email and try again.

**Comment by LFDanLu** [comment]
@drkstr101 Turns out that the script in question is actually dead code, so we've opted to remove it instead via https://github.com/adobe/react-spectrum/pull/2623. As a result, I'll be closing this PR but thanks for looking into this!

**Pull request closed without merging**